src/formatter.js

```diff
@@ -32,15 +32,21 @@ function escapeTeamCityString(str) {
     .replace(/\]/g, '|]');
 }
 
-var reportName = 'ESLint Violations';
-
 //------------------------------------------------------------------------------
 // Public Interface
 //------------------------------------------------------------------------------
-module.exports = function(results) {
+module.exports = function(results, _varNames) {
```

Comment:
I think that you need to give it a more descriptive name. `varNames` could mean anything. Maybe something like `reportNames` is better? (If I'm understanding correctly)